shell/common/shell.h

```diff
@@ -274,6 +274,13 @@ class Shell final : public PlatformView::Delegate,
   ///
   fml::WeakPtr<PlatformView> GetPlatformView();
 
+  //----------------------------------------------------------------------------
+  /// @brief      The IO Manager may only be accessed on the IO task runner.
+  ///
+  /// @return     A weak pointer to the IO manager.
+  ///
+  fml::WeakPtr<ShellIOManager> GetIOManager();
+
```

Comment:
I don't know if we care about adding this to the interface. I'm only using it for a test. It's also available through `UIDartSate::Current()`, but that requires getting isolate scope.